We assume the setup scripting will configure and mount the rootfs for us
so avoid running an rm -rf.

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

diff -ur oe-core.orig/meta/classes/image.bbclass oe-core/meta/classes/image.bbclass
--- oe-core.orig/meta/classes/image.bbclass	2013-01-22 22:26:29.000000000 -0600
+++ oe-core/meta/classes/image.bbclass	2013-01-22 22:53:40.584165930 -0600
@@ -183,7 +183,7 @@
 	#set -x
 	# When use the rpm incremental image generation, don't remove the rootfs
 	if [ "${INC_RPM_IMAGE_GEN}" != "1" -o "${IMAGE_PKGTYPE}" != "rpm" ]; then
-		rm -rf ${IMAGE_ROOTFS}
+		: # rm -rf ${IMAGE_ROOTFS}
 	elif [ -d ${T}/saved_rpmlib/var/lib/rpm ]; then
 		# Move the rpmlib back
 		if [ ! -d ${IMAGE_ROOTFS}/var/lib/rpm ]; then
